### PR TITLE
try to call TestCase methods

### PR DESCRIPTION
**Pull request by nickpell**

### CODE REVIEW

File: custom/ilsgateway/tests/test_reminders2.py [hunk 1]
```diff
@@ -172,6 +178,7 @@ def test_report_exclusion(self):
 class TestSupervisionStatusSet(RemindersTest):
 
     def setUp(self):
+        super(TestScript, super(ILSTestScript, super(RemindersTest, super(TestSupervisionStatusSet, self)))).setUp()
```
Comment (dimagimon) [inline custom/ilsgateway/tests/test_reminders2.py]: E501 line too long (116 > 115 characters)

File: custom/ilsgateway/tests/handlers/utils.py [hunk 2]
```diff
@@ -112,6 +114,7 @@ def tearDownClass(cls):
         SQLLocation.objects.all().delete()
         generator.delete_all_subscriptions()
         Domain.get_by_name(TEST_DOMAIN).delete()
+        super(TestScript, super(ILSTestScript, cls)).tearDownClass()
```
Comment (snopoke) [inline custom/ilsgateway/tests/handlers/utils.py]: woa, that's quite weird. Never seen that pattern before. What's the issue with calling the `ILSTestScript` superclass methods?

File: custom/ilsgateway/tests/test_reminders2.py [hunk 3]
```diff
@@ -16,13 +16,15 @@
 from custom.ilsgateway.tanzania.reminders.supervision import SupervisionReminder
 from custom.ilsgateway.tests.handlers.utils import ILSTestScript, TEST_DOMAIN, prepare_domain, create_products
 from custom.ilsgateway.utils import make_loc
+from custom.logistics.tests import TestScript
```
Comment (dimagimon) [inline custom/ilsgateway/tests/test_reminders2.py]: F401 'TestScript' imported but unused

File: custom/ilsgateway/tests/handlers/utils.py (hunk 2 above)
Comment (nickpell) [inline custom/ilsgateway/tests/handlers/utils.py]: some of the superclass methods do setup that conflicts with the subclass methods.  It's a sign that these tests are in need of serious refactoring, but I'm considering that out of scope for the django upgrade.

```super``` .... ```super``` (not surprisingly) didn't work, so I replaced it with https://github.com/dimagi/commcare-hq/pull/12135/commits/4d8c3cba1da378ff69dcb371de061f4adb3bd852

**Comment by nickpell** [comment]
✅ @emord cc: @snopoke

**Pull request merged**